sunpy/time/timerange.py

```diff
@@ -443,3 +443,25 @@ def __contains__(self, time):
         """
         this_time = parse_time(time)
         return this_time >= self.start and this_time <= self.end
+
+    def intersects(self, other):
+        """
+        Return `True` if this interval overlaps with *other*.
+
+        Parameters
+        ----------
+        other : sunpy.time.TimeRange
+
```

Comment:
```suggestion
		The other time range to check for intersection.
```
Kind of redundant but I don't really like blank parameter descriptions.